macOS release does not work

macOS release binary does not work on macOS 10.15.7 (Catalina)

Not sure whether it is expected to work or not and for which version of macOS the build is intended for - but I just wanted to share this as maybe someone else may experience this error as well.

```
$ tar xfv x86_64-apple-darwin14-mklittlefs-47e8167.tar.gz
x mklittlefs/
x mklittlefs/mklittlefs
$ cd mklittlefs
$ ./mklittlefs
[1]    33495 abort      ./mklittlefs
```

Building from the source fortunately works without any issue
```
$ make dist
c++ -std=gnu++11 -Os -Wall   -Itclap -Iinclude -Ilittlefs -I. -D VERSION=\"0.2.3-31-g295fe9b\" -D LITTLEFS_VERSION=\"v2.0.5\" -D BUILD_CONFIG=\"\" -D BUILD_CONFIG_NAME=\"-generic\" -D __NO_INLINE__ -D LFS_NAME_MAX=32   -c -o main.o main.cpp
cc -std=gnu99 -Os -Wall   -Itclap -Iinclude -Ilittlefs -I. -D VERSION=\"0.2.3-31-g295fe9b\" -D LITTLEFS_VERSION=\"v2.0.5\" -D BUILD_CONFIG=\"\" -D BUILD_CONFIG_NAME=\"-generic\" -D __NO_INLINE__ -D LFS_NAME_MAX=32   -c -o littlefs/lfs.o littlefs/lfs.c
cc -std=gnu99 -Os -Wall   -Itclap -Iinclude -Ilittlefs -I. -D VERSION=\"0.2.3-31-g295fe9b\" -D LITTLEFS_VERSION=\"v2.0.5\" -D BUILD_CONFIG=\"\" -D BUILD_CONFIG_NAME=\"-generic\" -D __NO_INLINE__ -D LFS_NAME_MAX=32   -c -o littlefs/lfs_util.o littlefs/lfs_util.c
c++ main.o littlefs/lfs.o littlefs/lfs_util.o -o mklittlefs
strip mklittlefs
cp mklittlefs mklittlefs-0.2.3-31-g295fe9b-generic-osx/
tar czf mklittlefs-0.2.3-31-g295fe9b-generic-osx.tar.gz mklittlefs-0.2.3-31-g295fe9b-generic-osx
make: *** No rule to make target `test', needed by `dist'.  Stop.

$ cd mklittlefs-0.2.3-31-g295fe9b-generic-osx
$ ./mklittlefs
PARSE ERROR:
             Required arguments missing: list, unpack, create, image_file

Brief USAGE:
   ./mklittlefs  {-c <pack_dir>|-u <dest_dir>|-l} [-d <0-5>] [-a] [-b
                 <number>] [-p <number>] [-s <number>] [--] [--version]
                 [-h] <image_file>

For complete USAGE and HELP type:
   ./mklittlefs --help
```

--- a/Makefile
+++ b/Makefile
@@ -87,10 +87,6 @@ all: $(TARGET)
 
 dist: $(DIST_ARCHIVE)
 
-ifndef SKIP_TESTS
-dist: test
-endif
-
 $(DIST_ARCHIVE): $(TARGET) $(DIST_DIR)
 	cp $(TARGET) $(DIST_DIR)/
 	$(ARCHIVE_CMD) $(DIST_ARCHIVE) $(DIST_DIR)

